Jenkinsfile

```diff
@@ -17,6 +17,9 @@
  */
 
 properties([
+    // For master keep all builds for one year
+    // For other branches/PRs keep at most 5 builds for at most two months
+    buildDiscarder(logRotator(daysToKeepStr: env.BRANCH_NAME=='master'?'365':'60', numToKeepStr: env.BRANCH_NAME=='master'?'':'5')),
```

Comment:
instead of 2 `?:` ; why not have a full `if` ? more readble; and then the comment could be in the branches too - or can be even relaxed

Reply:
@kgyrtkirk Is the refactoring below what you had in mind?
```groovy
def discardBuildsDaysLimit = '365'
def discardBuildsEntriesLimit = ''
if (env.BRANCH_NAME != 'master') {
  discardBuildsDaysLimit = '60'
  discardBuildsEntriesLimit = '5'
}
properties([
    buildDiscarder(logRotator(daysToKeepStr: discardBuildsDaysLimit, numToKeepStr: discardBuildsEntriesLimit))
```

Reply:
Since you approved the initial patch I plan to merge that one in a couple of hours. Let me know if you have other comments in the meantime.

Reply:
I think above one tells the story better; following the names of the exising api is also beneficial; `discardDaysToKeep` /  `discardNumToKeep`

but I've no strong opinion - this is a rarely changing thing...so it can be anyhow if it works as intended...